examples/shouldi/tests/test_cargo_audit.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
+
+from dffml.util.os import prepend_to_path
+from dffml.util.net import cached_download_unpack_archive
+from dffml.util.asynctestcase import AsyncTestCase
+
+from shouldi.cargo_audit import run_cargo_audit
+
+
+class TestRunCargo_AuditOp(AsyncTestCase):
+    @cached_download_unpack_archive(
+        "https://static.rust-lang.org/dist/rust-1.42.0-x86_64-unknown-linux-gnu.tar.gz",
+        pathlib.Path(__file__).parent / "cargo_audit.tar.gz",
+        pathlib.Path(__file__).parent / "cargo-audit-download",
```

Comment:
It looks like you've got the downloads for rust and tarpaulin, but you've named the rust download `cargo_audit` and don't have the download for `cargo_audit` itself.